Modules/Filtering/FFT/src/itkFFTWGlobalConfiguration.cxx

```diff
@@ -157,15 +157,9 @@ FFTWGlobalConfiguration::GetInstance()
       m_PimplGlobals->m_Instance = Self::New();
       if (!m_PimplGlobals->m_Instance)
       {
```

Comment:
Is it really necessary to manually check the result of `Self::New()` here? I mean, can't we just assume that Self::New() _itself_ throws an exception when it fails to create an FFTWGlobalConfiguration instance?